pkg/sql/catalog/descs/txn.go

```diff
@@ -75,13 +77,13 @@ func (cf *CollectionFactory) Txn(
 	for {
 		var modifiedDescriptors []lease.IDVersion
 		var deletedDescs catalog.DescriptorIDSet
-		var descsCol Collection
+		var descsCol *Collection
```

Comment:
why did you make this change?

Reply:
It's because `descsCol` will take the return value from `cf.MakeCollection()` below , which I changed the `cf.MakeCollection()` to return a pointer, and as you said, this is incorrect.